library/sqsdevCreds.properties

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
We will implement a new "credentials store" in a separate JIRA. For now I have moved the location of the properties files to under test/resources .